python/README.md

```diff
@@ -0,0 +1,185 @@
+[![Build Status](https://travis-ci.com/pulumi/pulumi-eks.svg?token=eHg7Zp5zdDDJfTjY8ejq&branch=master)](https://travis-ci.com/pulumi/pulumi-eks)
```

Comment:
A lot of this readme is specific to Node

Reply:
Agreed. The codegen just pulls in the root README.md. I opened https://github.com/pulumi/pulumi-eks/issues/471 to track updating it.